### PR TITLE
add resource links

### PR DESCRIPTION
**Pull request by lindalu-MSFT**
Per bug [3993611](https://office.visualstudio.com/OC/_workitems/edit/3993611)

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[fd10e10](https://github.com/OfficeDev/office-js-docs-pr/commits/fd10e10ce78b7074906595e8b2329cf160acab1e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-update-addin-hub-resources-3993611/docs/index.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/index?branch=pr-en-us-2017) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202007132317413006-2017%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=uZZo%2BqpmGyBVPVoQX6zG9aWS52xZN1xKQVHgzsyfVAs%3D&st=2020-07-13T23%3A14%3A00Z&se=2020-08-13T23%3A19%3A00Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by ElizabethSamuel-MSFT** [comment]
@lindalu-MSFT Should we also add a link to the resources page: https://docs.microsoft.com/office/dev/add-ins/resources/resources-links-help?

**Comment by lindalu-MSFT** [comment]
Thanks @ElizabethSamuel-MSFT, good idea and I added this to the bottom of the Resources tab.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[28872c2](https://github.com/OfficeDev/office-js-docs-pr/commits/28872c22369087276c2051a6ed16fa52a7c10330)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-update-addin-hub-resources-3993611/docs/index.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/index?branch=pr-en-us-2017) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C14%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202007141823374320-2017%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=DkQYqcooqCyTEUjVmhFAArfQxEsGSsoOEk5TLEgFHfo%3D&st=2020-07-14T18%3A19%3A55Z&se=2020-08-14T18%3A24%3A55Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/index.yml [hunk 1]
```diff
@@ -96,3 +96,9 @@ landingContent:
             url: https://officespdev.uservoice.com/
           - text: COM or VSTO add-ins documentation
             url: /office/client-developer/office-client-development
+          - text: Office Add-ins PnP samples repo
+            url: https://github.com/OfficeDev/PnP-OfficeAddins
+          - text: Office Add-ins blogs
+            url: https://developer.microsoft.com/en-us/office/gallery/?filterBy=Add-ins,Blog%20Posts
```
Comment (ElizabethSamuel-MSFT) [inline docs/index.yml:102]: Do we have a locale agnostic version of this link (i.e. without en-us)?

Comment (lindalu-MSFT) [reply docs/index.yml:102]: Yasssssssss ooops! My bad. Fixed both URLS!

**Pull request merged**